config/ie/buildAll

```diff
@@ -81,13 +81,13 @@ if IEEnv.platform() in ( "cent7.x86_64", ) :
 	for aiVersion in IEEnv.activeVersions( IEEnv.registry["apps"]["arnold"] ):
 
 		for mayaVersion in IEEnv.activeAppVersions( "maya" ) :
-			build( [ "APP=maya", "APP_VERSION="+mayaVersion, "ARNOLD_VERSION=" + aiVersion ] )
+			build( [ "APP=maya", "APP_VERSION="+mayaVersion, "ARNOLD_VERSION=" + aiVersion, "APPLESEED_VERSION=UNDEFINED" ] )
```

Comment:
we seem to have been successfully building Appleseed for the DCCs variations so far. Does it fail to compile now?

Reply:
Yeah, lets just use the last one like we do for gaffer: https://github.com/GafferHQ/gaffer/blob/master/config/ie/installAll#L23

Reply:
> Yeah, lets just use the last one like we do for gaffer: https://github.com/GafferHQ/gaffer/blob/master/config/ie/installAll#L23

Updated in  dffbf7a